pkg/kwok/controllers/node_controller.go

```diff
@@ -277,6 +284,7 @@ func (c *NodeController) watchResources(ctx context.Context, opt metav1.ListOpti
 					node := event.Object.(*corev1.Node)
 					if c.need(node) {
 						c.putNodeInfo(node)
+						c.applyMetrics(ctx, node)
```

Comment:
Should we delete the registry when the node is deleted, or should we set values to 0?